### PR TITLE
Bsd 1516 rdf dump generator

### PR DESCRIPTION
**Pull request by dipayan1985**

### CODE REVIEW

File: utils/bioschemas-rest-client/src/main/java/uk/ac/ebi/biosamples/utils/BioSchemasJsonLdWriter.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+import com.mongodb.MongoClientURI;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+public class BioSchemasJsonLdWriter {
+    private static final String LDJSON = ".ldjson";
+    private static final String BIOSAMPLES_BASE_URI = "https://www.ebi.ac.uk/biosamples/samples/";
+    private static final String BIOSAMPLES = "biosamples";
+    private static final String MONGO_SAMPLE = "mongoSample";
+    private static final String MONGO_DB_CONNECT_STRING = "mongodb://biosd:TkL51qjIt81aKQ@mongodb-hxvm-009.ebi.ac.uk:27017," +
```
Comment (theisuru) [inline utils/bioschemas-rest-client/src/main/java/uk/ac/ebi/biosamples/utils/BioSchemasJsonLdWriter.java]: I think we should get this from config or people from outside can see this, we might need to do a rebase to remove this from commit history

File: utils/bioschemas-rest-client/src/main/java/uk/ac/ebi/biosamples/utils/BioSchemasRdfGenerator.java [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Scanner;
+import java.util.function.Consumer;
+
+public class BioSchemasRdfGenerator {
+    private static final String LDJSON = ".ldjson";
+    private static final String BIOSAMPLES_BASE_URI = "https://www.ebi.ac.uk/biosamples/samples/";
+    private static final String MONGO_DB_CONNECT_STRING = "mongodb://biosd:TkL51qjIt81aKQ@mongodb-hxvm-009.ebi.ac.uk:27017," +
```
Comment (theisuru) [inline utils/bioschemas-rest-client/src/main/java/uk/ac/ebi/biosamples/utils/BioSchemasRdfGenerator.java]: Same here

File: webapps/core/src/main/java/uk/ac/ebi/biosamples/controller/SampleRestController.java [hunk 3]
```diff
@@ -95,6 +107,41 @@ public SampleRestController(SampleService sampleService,
...
+		}
+
+		bioSamplesAapService.checkAccessible(sample.get());
+
+		// TODO cache control
+		return new ResponseEntity<String>(
+				sampleResourceAssembler.toResource(sample.get(), decodedLegacyDetails, decodedCurationDomains).getContent().getAccession(),
+				HttpStatus.OK);
+	}
+
```
Comment (theisuru) [inline webapps/core/src/main/java/uk/ac/ebi/biosamples/controller/SampleRestController.java]: Is this also related to rdf dump generator

File: utils/bioschemas-rest-client/src/main/java/uk/ac/ebi/biosamples/utils/BioSchemasJsonLdWriter.java (hunk 1 above)
Comment (dipayan1985) [reply utils/bioschemas-rest-client/src/main/java/uk/ac/ebi/biosamples/utils/BioSchemasJsonLdWriter.java]: Yes right. We will need a rebase. I will remove this.

File: webapps/core/src/main/java/uk/ac/ebi/biosamples/controller/SampleRestController.java (hunk 3 above)
Comment (dipayan1985) [reply webapps/core/src/main/java/uk/ac/ebi/biosamples/controller/SampleRestController.java]: No this is from the other pull request.

**Pull request merged**